SymPEP-0001.md

```diff
@@ -0,0 +1,222 @@
+SymPEPs generally live in a pull request.
+
+Eventually, after discussion, there may be a consensus that the SymPEP should
+be accepted–see the next section for details. At this point the status
+becomes **Accepted**.
+
+Once a SymPEP has been **Accepted**, the reference implementation must be
+completed. When the reference implementation is complete and incorporated into
+the main source code repository, the status will be changed to **Final**.
+
+A SymPEP can also be assigned status **Deferred**. The SymPEP author or a core
+developer can assign the SymPEP this status when no progress is being made on
+the SymPEP.
+
+A SymPEP can also be **Rejected**. Perhaps after all is said and done it was
+not a good idea. It is still important to have a record of this fact. The
+**Withdrawn** status is similar—it means that the SymPEP author themselves has
+decided that the SymPEP is actually a bad idea, or has accepted that a
+competing proposal is a better alternative.
+
+SymPEPs can also be **Superseded** by a different SymPEP, rendering the
+original obsolete.
+
+Process SymPEPs may also have a status of **Active** if they are never meant
+to be completed, e.g. SymPEP 1 (this SymPEP).
+
+### Merging the SymPEP Document Pull Request
+
+Whenever a SymPEP moves from the **Draft** status to one of the other above
+statuses, the header should be updated, and the corresponding pull request
+should be merged. This way the document lives inside of the SymPEP repository
+proper. SymPEPs that are merged are not set in stone, and may be updated by
+future pull requests (although SymPEPs that are **Accepted** should generally
+not be significantly modified once they have reached that status). The purpose
+of merging is simply to make the SymPEP visible in the repository, even if it
+isn't yet accepted or rejected. SymPEP discussions that have stalled should
+also be merged, so that the SymPEP becomes visible in the SymPEP repository
+proper—again, discussion may be picked up again with a new pull request.
+
+## Accepting a SymPEP
+
+Once a SymPEP is Accepted by consensus of all interested contributors, an
+email should be sent to the [SymPy mailing
+list](http://groups.google.com/group/sympy) with a subject like:
+
+    Proposal to accept SymPEP #<number>: <title>
+
+In the body of your email, you should:
+
+- link to the latest version of the SymPEP,
+
+- briefly describe any major points of contention and how they were resolved,
+
+- include a sentence like: “If there are no substantive objections within 7
+  days from this email, then the SymPEP will be accepted; see SymPEP 1 for
+  more details.”
+
+After you send the email, add the the email thread to the Discussion section
+of the SymPEP, so that people can find it later.
+
+Generally the SymPEP author will be the one to send this email, but anyone can
+do it – the important thing is to make sure that everyone knows when a SymPEP
+is on the verge of acceptance, and give them a final chance to respond. If
```

Comment:
In line 172, it is said that the email is sent after acceptance, yet here it says it is on the verge of acceptance. Those seem inconsistent.

Reply:
I think it means that when the contributors who are actively participating in the SymPEP's PR have come to a conclusion that the proposal is ready to be sent to the general public(the mailing list). But yeah, the wording does seem inconsistent and kind of confusing. 

There are two different processes involved here
- having the approval of "contributors"
- having the approval of the "community"
Maybe we should not call both "acceptance"; this will get confusing

Reply:
That makes sense to me.